tests/test_python-opensubtitles.py

```diff
@@ -0,0 +1,124 @@
+import pytest
+from os import path
+
+
+class TestPythonOpenSubtitles(object):
+    @staticmethod
+    @pytest.fixture
+    def data():
+        class Data:
+            username = 'doctest'
+            password = 'doctest'
+            name = 'Dark City'
+            path = '/home/alex/Videos/Dark.City.1998.Directors.Cut.BRRip.H264.AAC.5.1ch.Gopo/'
+            video = 'Dark.City.1998.Directors.Cut.BRRip.H264.AAC.5.1ch.Gopo.mp4'
+            subtitle = 'Dark.City.1998.Directors.Cut.BRRip.H264.AAC.5.1ch.Gopo.srt'
+        return Data
+
+    @staticmethod
+    @pytest.fixture
+    def os():
+        from pythonopensubtitles.opensubtitles import OpenSubtitles
```

Comment:
You are using this fixture in all the methods, maybe you could just import this at the beginning of the file.